app/templates/frameworks/_framework_actions.html

```diff
@@ -24,128 +24,80 @@
-      <p class="browse-list-item-status-title">You’re submitting</p>
-      {% else %}
-    <div class="browse-list-item-status-default">
-      <p class="browse-list-item-status-title">You’ve completed:</p>
-      {% endif %}
-      <ul class="browse-list-item-status-list">
-        {% for lot in completed_lots %}
-          {% if lot.one_service_limit %}
-        <li>
-            {{lot.name|lower}}
-        </li>
+    </li>
+    <li class="dm-task-list-item">
+      <span class="app-task-list__task-name">
+        {% if application_company_details_confirmed %}
+          {% if declaration_status == 'unstarted' %}
+            <a href="{{ url_for('.framework_start_supplier_declaration', framework_slug=framework.slug) }}">Make your supplier declaration</a>
           {% else %}
-        <li>
-          {{ lot.complete_count }}
-          {{ lot.unitSingular if (1 == lot.complete_count) else lot.unitPlural }} in
-          {{ lot.name|lower }}
-        </li>
+            <a href="{{ url_for('.framework_supplier_declaration_overview', framework_slug=framework.slug) }}">Make your supplier declaration</a>
           {% endif %}
-        {% endfor %}
-      </ul>
-    </div>
-    {% endif %}
-  {% endif %}
-</li>
+        {% else %}
+          Make your supplier declaration
+        {% endif %}
+      </span>
+      {% if not application_company_details_confirmed %}
+        <strong class="dm-task-list-indicator-not-completed" id="dm-declaration-cantstart">Can't start yet</strong>
+      {% elif declaration_status == 'unstarted' %}
+        <strong class="dm-task-list-indicator-not-completed" id="dm-declaration-todo">To do</strong>
+      {% elif declaration_status == 'started' %}
+        <strong class="dm-task-list-indicator-not-completed" id="dm-declaration-inprogress">In progress</strong>
+      {% elif declaration_status == 'complete' %}
+        <strong class="dm-task-list-indicator-completed" id="dm-declaration-done">Done</strong>
+      {% endif %}
+    </li>
+    <li class="dm-task-list-item">
+
+        {% if application_company_details_confirmed %}
+        <a href="{{ url_for('.framework_submission_lots', framework_slug=framework.slug) }}">
+          Add, edit and complete services
+        </a>
+        {% else %}
+          Add, edit and complete services
+        {% endif %}
+
+      {% if not application_company_details_confirmed %}
+        <strong class="dm-task-list-indicator-not-completed" id="dm-services-cantstart">Can't start yet</strong>
+      {% elif not counts.draft and not counts.complete %}
+        <strong class="dm-task-list-indicator-not-completed" id="dm-services-todo">To Do</strong>
+      {% elif counts.draft %}
+        <strong class="dm-task-list-indicator-not-completed" id="dm-services-inprogress">In progress</strong>
+      {% else %}
+        <strong class="dm-task-list-indicator-completed" id="dm-services-done">{{ counts.complete }} {{ pluralize(counts.complete, 'service', 'services') }}</strong>
```

Comment:
Like this. Showing them how many are complete. Would be confusing to do something like:
`{{ if counts.complete }}Done{{ endif }}`

:100:

Reply:
Not sure - I was going by Joe's proposals.